electron/packetDefs.js

```diff
@@ -257,8 +257,15 @@ const INBOUND_PACKET_DEFS = {
     ['fastLoadCell2', asFloat],
     // ['fastLoadCellSum', asFloat],
   ],
-  220: [
-    ['capacitor', asFloat],
+  221: [
+    ['capVal', asFloat],
+    ['capValFiltered', asFloat],
+    ['capTemperature', asFloat],
+  ],
+  222: [
+    ['capVal', asFloat],
+    ['capValFiltered', asFloat],
+    ['capTemperature', asFloat],
```

Comment:
Why are there two different packets here?

Reply:
Not sure, @xy3710’s doing